serverlessworkflow/modules/ROOT/pages/cloud/build-and-deploy-with-serverless-operator-on-kubernetes.adoc

```diff
@@ -0,0 +1,175 @@
+:kogitogreetinexample_url: https://github.com/kiegroup/kogito-examples/tree/stable/serverless-workflow-examples/serverless-workflow-greeting-quarkus
+:kanikoissue_url: https://github.com/GoogleContainerTools/kaniko/issues/2201
+
+This document describes how to build and deploy your workflow application using a local Kubernetes cluster, such as link:{minikube_url}[Minikube], along with the link:{kogitoserverlesosperator_url}[{product_name} Serverless Operator].
+
+Using the {product_name} Serverless Operator, you will be able both to build and deploy a {product_name} {context} application so that you only need the workflow definition.
+
+Suppose you already have a container built and pushed on a container registry and you want to deploy it on the Kubernetes cluster. In that case, you can do it without the operator following the guide xref:cloud/deploying-on-kubernetes.adoc[Deploying your {context} application on Kubernetes].
+
+The link:{kogitoserverlesosperator_url}[{product_name} Serverless Operator] is currently in an alpha version and under active development and, at the moment, is supporting {context} definitions that are using:
+
+* Functions
+* States
+    - Supported State types:
+    - Switch including dataConditions
+    - Inject including data with a transition
+* Operation including Actions with functionRef with arguments
+* KeepActive
+* AutoRetries
+* ExpressionsLang (jq or jsonpath)
+
+.Prerequisites
+* A workflow definition.
+* A Kubernetes cluster with admin privileges (if you haven't got one prepare and use a local Minikube instance).
+* `kubectl` command-line tool is installed. Otherwise, Minikube handles it.
+
+.Prepare a Minikube instance
+
+[source,shell,subs="attributes+"]
+----
+minikube start --cpus 4 --memory 4096 --addons registry --addons metrics-server --insecure-registry "10.0.0.0/24" --insecure-registry "localhost:5000"
+----
+
+[NOTE]
+====
+To speed up, you can increase cpus and memory options. For example, use `--cpus 12 --memory 16384`.
+====
+
+[TIP]
+====
+If it does work with the default driver, aka `docker`, you can try to start with the `podman` driver:
+====
+
+[source,shell,subs="attributes+"]
+----
+minikube start [...] --driver podman
+----
+
+[IMPORTANT]
+====
+There are some issues with the `crio` container runtime and Kaniko that the operator is using. Reference: link:{kanikoissue_url}[ISSUE-2201]
+====
+
+.Setup {product_name} Serverless Operator
+
+In order to have an up and running instance of the {product_name} Serverless Operator you can use the following command:
+
+[source,shell,subs="attributes+"]
+----
+kubectl create -f https://raw.githubusercontent.com/kiegroup/kogito-serverless-operator/main/operator.yaml
+----
+
+[NOTE]
+====
+Follow the deployment of the {product_name} Serverless Operator:
+====
++
+[source,shell,subs="attributes+"]
+----
+kubectl get pod -n kogito-serverless-operator-system --watch
+----
+
+[TIP]
```

Comment:
In my opinion, we can avoid using TIP. Can you please let me know if these are optional steps?
```suggestion
You can also follow the operator’s log:
+
[source,shell,subs="attributes+"]
----
kubectl logs deployment/kogito-serverless-operator-controller-manager -n kogito-serverless-operator-system -f
----
```